Spatial.gd

```diff
@@ -929,7 +935,14 @@ func _on_NewIcon_pressed():
 	position.set_x(self.player_position.x)
 	position.set_y(self.player_position.y)
 	position.set_z(-self.player_position.z)
-	gen_new_icon(self.next_texture_trail, waypoint_icon, self.currently_active_category)
```

Comment:
wat
https://github.com/AsherGlick/Burrito/pull/318/files#diff-de757fecd7208a343c7d1d1ac26b99a2a87d2f8f61537b01a868f0e058441bbcR932
How did this pass testing in the last PR?

Reply:
Not sure but it's being deleted.